.github/workflows/linux-eic-shell.yml

```diff
@@ -285,6 +285,43 @@ jobs:
         path: sim_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4hep.root
         if-no-files-found: error
 
+  eicrecon-benchmarks-plugins:
+    runs-on: ubuntu-latest
+    needs:
+    - build
+    - ddsim-gun
+    strategy:
+      matrix:
+        CC: [gcc]
+        particle: [e]
+        detector_config: [brycecanyon]
+        benchmark_plugins:
+        - EcalBarrelScFiCheck
+        - femc_studies
+        - lfhcal_studies
+        - tof_efficiency
+        - TRACKINGcheck
+        - tracking_efficiency
+        - tracking_occupancy
+    steps:
+    - uses: actions/download-artifact@v3
+      with:
+        name: install-${{ matrix.CC }}-eic-shell-Release
+    - uses: actions/download-artifact@v3
+      with:
+        name: sim_${{ matrix.particle }}_1GeV_20GeV_${{ matrix.detector_config }}.edm4hep.root
+    - uses: cvmfs-contrib/github-action-cvmfs@v3
+    - name: Run EICrecon
+      uses: eic/run-cvmfs-osg-eic-shell@main
+      with:
+        platform-release: "jug_xl:nightly"
+        setup: /opt/detector/setup.sh
+        run: |
+          export DETECTOR_CONFIG=${DETECTOR}_${{ matrix.detector_config }}
+          export JANA_PLUGIN_PATH=$PWD/lib/EICrecon/plugins${JANA_PLUGIN_PATH:+:${JANA_PLUGIN_PATH}}
+          chmod a+x bin/*
+          $PWD/bin/eicrecon -Pplugins=${{ matrix.benchmark_plugins }} -Ppodio:output_file=rec_${{ matrix.particle }}_1GeV_20GeV_${{ matrix.detector_config }}.edm4eic.root sim_${{ matrix.particle }}_1GeV_20GeV_${{ matrix.detector_config }}.edm4hep.root -Pplugins=dump_flags,janadot -Pdump_flags:json=${{ matrix.particle }}_${{ matrix.detector_config }}_flags.json -Pjana:warmup_timeout=0 -Pjana:timeout=0
+
```

Comment:
No artifacts produced?

Reply:
It wasn't really the intention to start looking at these artifacts, but we could store them. I am not sure if they contain anything actionable, but it is a good test to make sure they are actually written.